vignettes/IntroToAnnotationPackages.Rmd

```diff
@@ -0,0 +1,383 @@
+---
+title: "AnnotationDbi: Introduction To Bioconductor Annotation Packages"
+author: 
+  - name: "Marc Carlson"
+  - name: "Beryl Kanali"
+    affiliation: "Converted vignette from Sweave to Rmarkdown"
+date: "Last modified: November 2022; Compiled: `r format(Sys.time(), '%B %d , %Y')`"
+
+vignette: >
+  %\VignetteIndexEntry{1. Introduction To Bioconductor Annotation Packages}
+  %\VignetteDepends{org.Hs.eg.db, TxDb.Hsapiens.UCSC.hg19.knownGene, EnsDb.Hsapiens.v75}
+  %\VignetteKeywords{annotation, database}
+  %\VignettePackage{AnnotationDbi}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+output:
+  BiocStyle::html_document
+---
+
+```{r  label = fig:dbtypes, echo = FALSE, fig.align="center", fig.cap = "Annotation Packages: the big picture", out.width = '60%'}
+knitr::include_graphics("databaseTypes.png")
+```
+
+```{r echo=FALSE, include=FALSE}
+if (!require("BiocManager", quietly = TRUE))
+    install.packages("BiocManager")
+
+BiocManager::install("BiocStyle", force = TRUE)
+```
+
+
+```{r include=FALSE}
+library(knitr)
+opts_chunk$set(tidy=FALSE)
+```
+
+
+```{r include=FALSE}
+library(BiocStyle)
+```
```

Comment:
We don't need these lines 24-40, I Remember that One of the requirements of a vignette is that it must not install anything without the user's knowledge. Also BiocStyle and knitr are included in suggests.